pinot-integration-tests/pom.xml

```diff
@@ -249,5 +249,16 @@
       <artifactId>annotations</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+        <groupId>org.apache.spark</groupId>
```

Comment:
The format is not correct. I've found some formatting issue in other changes as well. Can you apply `Pinot Style` and reformat the changes